### PR TITLE
chore: release case / task during assignment when no employee is selected

### PR DESCRIPTION
**Pull request by felixcicatt**
Release case / task during assignment when no employee is selected
Solves PZ-2217

### CODE REVIEW

File: src/main/kotlin/net/atos/zac/zaak/ZaakService.kt [hunk 1]
```diff
@@ -66,7 +66,7 @@ class ZaakService @Inject constructor(
                         bepaalRolMedewerker(it, zaak),
                         explanation
                     )
-                }
+                } ?: zrcClientService.deleteRol(zaak, BetrokkeneType.MEDEWERKER, explanation)
```
Comment (edgarvonk) [inline src/main/kotlin/net/atos/zac/zaak/ZaakService.kt:69]: Add a unit test for this scenario? We have quite some test coverage already so should be quite easy.

Comment (edgarvonk) [inline src/main/kotlin/net/atos/zac/zaak/ZaakService.kt:69]: Checking: are you sure it should be this betrokkene type?

**Review by edgarvonk — CHANGES_REQUESTED** [review]

File: src/main/kotlin/net/atos/zac/zaak/ZaakService.kt (hunk 1 above)
Comment (felixcicatt) [reply src/main/kotlin/net/atos/zac/zaak/ZaakService.kt:70]: I think so, it's the same role we use in the releasement function

File: src/test/kotlin/net/atos/zac/task/TaskServiceTest.kt [hunk 2]
```diff
@@ -93,7 +93,7 @@ class TaskServiceTest : BehaviorSpec({
         every { eventingService.send(capture(screenEventSlot)) } just runs
         every { indexeerService.indexeerDirect(restTaakToekennenGegevens.taakId, ZoekObjectType.TAAK, true) } just runs
 
-        When("the 'assign task' function is called with REST taak toekennen gegevens with a group and user") {
+        When("the 'assign task' function is called with REST taak toekennen gegevens with a group and WITHOUT a user") {
```
Comment (edgarvonk) [inline src/test/kotlin/net/atos/zac/task/TaskServiceTest.kt:96]: No need to shout haha. ;-)

**Review by edgarvonk — APPROVED** [review]
nice one

**Pull request merged**